docker/tds/docker-compose.yml

```diff
@@ -94,6 +94,12 @@ services:
     environment:
       - JAVA_OPTS=${CONTAINER_JAVA_OPTS}
 
+  #TDS Equation Scoring Engine
+  equation-scoring-image:
```

Comment:
nit: seems like the others don't use `-image` as the suffix since image is the next level.